eo-maven-plugin/src/main/java/org/eolang/maven/objectionary/ObjsDefault.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2023 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven.objectionary;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.eolang.maven.hash.CommitHash;
+
+/**
+ * Default objectionaries.
+ * @since 0.29.6
+ */
+public final class ObjsDefault implements Objectionaries {
+    /**
+     * Hash-map.
+     */
+    private final Map<String, Objectionary> map;
+
+    /**
+     * Ctor.
+     */
+    public ObjsDefault() {
+        this(new HashMap<>());
+    }
+
+    /**
+     * Ctor.
+     * @param ojs Objectionaries hash-map.
+     */
+    ObjsDefault(final Map<String, Objectionary> ojs) {
+        this.map = ojs;
+    }
+
+    @Override
+    public Objectionaries with(final CommitHash hash, final Objectionary objectionary) {
+        this.map.putIfAbsent(hash.value(), objectionary);
```

Comment:
@maxonfjvipon Maybe it's better to overwrite values when we call `with(...)` all the time? Otherwise it could be misleading

Reply:
@volodya-lombrozo I don't think so, at least for now, when `Objectionaries` is used only by `ProbeMojo` and `PullMojo` where it should not be overwritten. Otherwise we could you just a simple `HashMap` instead of `Objectionaries`

Reply:
@maxonfjvipon Right now we have the next problem:
1. I create `Objectionaries`
2. Call `.with(First Objectionary)`
3. Call `.with(Second Objectionary)`
4. call `.get()` and receive the `First Objectionary`. 

Again the design is misleading. I'm like a developer expect to get `Second Objectionary`. In other words, you just swallow `Second Objectionary` for now